static/ankersrv.js

```diff
@@ -29,8 +29,34 @@ $(function () {
     } else {
         /* Clipboard support missing: remove clipboard icons to minimize confusion */
         $("[data-clipboard-src]").remove();
-    };
+    }
+
+    /**
+     * Converts a string to its boolean value.
+     *
+     * @function
+     * @param {string} string - The string to be converted.
+     * @returns {boolean} - True if the input string is "true", "yes", or "1"; otherwise, false.
+     */
+    function stringToBoolean(string) {
+        switch (string.toLowerCase().trim()) {
+            case "true":
+            case "yes":
+            case "1":
+                return true;
+            default:
+                return false;
+        }
+    }
 
+    // Get URL parameter for fullscreen and apply it if needed, this emulates fullscreen
+    let urlParams = new URLSearchParams(window.location.search);
+    let fullscreenParam = stringToBoolean(urlParams.get("fullscreen"));
```

Comment:
This breaks the js if `fullscreen` arg is not given - "string is null" error in `stringToBoolean`

Reply:
✅